src/olympia/shelves/tests/test_views.py

```diff
@@ -223,9 +213,9 @@ def test_full_response(self):
 
         self.hpshelf_a.update(enabled=True)
 
```

Comment:
Can you also test that pagination stuff isn't in this response ?

Reply:
https://github.com/mozilla/addons-server/pull/16370/commits/06d05bc2ee189cf6bf953fb48aeac65ea2c19003